src/components/SMI-Table/LandscapeTable.style.js

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+
+export const TableWrapper = styled.div`
+overflow-x: auto;
+
+table {
+    border-spacing: 0;
+    width: 100%;
+  
+    td {
+        padding: 5px;
+        height: 60px;
```

Comment:
@dhruv0000 could you please update the CSS to use 'rem' unit instead of 'px'. Throughout the changes only you have done?

Reply:
Sure. But you might want to also look at the `Landscape-Table` CSS, on top of which I made some additional changes.

Reply:
Having said that, I don't think the modification I made on top of the Landsacpe-Table CSS had used 'px' anywhere.

Reply:
Ohh, I missed that in my PR. If you don't mind could you please update the landscape table css as well to use 'rem' instead of 'px' ?

Also, then the css for both the tables would be same and the same file can be used for both instead of creating another one.